### ISSUE TITLE
Continue searching backend even if a trace is found in the ingesters

### ISSUE THREAD

**Issue by joe-elliott** [grafana/tempo#344]
**Is your feature request related to a problem? Please describe.**
As part of our plans to support "exhaustive search" for traces let's search the backend even if the ingesters return a result.  This is related to #161.

**Describe the solution you'd like**
Continue searching the backend even if the ingesters succeed and combine the results.

**Additional context**
@annanay25 we need to consider this as we move forward with the query frontend.  I'm thinking we will add a number of parameters to the `GET /api/traces/xxxxxx` endpoint that the query frontend can use for sharding like:

```
GET /api/traces/xxx?startblock=xxx&endblock=xxx&ingesters=false
```

By default the `/api/traces/xxx` endpoint should just search everything.

**Comment by annanay25** [comment]
> I'm thinking we will add a number of parameters to the GET /api/traces/xxxxxx endpoint that the query frontend can use for sharding

Precisely what I was thinking as well. I'll have something more concrete to share soon.

**Comment by joe-elliott** [comment]
Fixed in #489
